content/basics/templates/intrinsic-functions/index.md

```diff
@@ -58,6 +58,35 @@ your template more flexible. Let's convert `AmiID` to variable and pass it to re
           InstanceType: !Ref InstanceType
     ```
 
+#### Ref vs Fn::Sub
+
+The intrinsic function Ref returns the value of the specified parameter or resource. When you are declaring a resource in a template and you need to specify another template resource by name, you can use the Ref to refer to that other resource. In general, Ref returns the name of the resource.
+
+Syntax for JSON template
+```
+{ "Ref" : "logicalName" }
+```
```

Comment:
We do not use JSON syntax through the lab and none of the other labs show examples of the JSON syntax. For consistency, i would suggest to remove this.

Reply:
Removed the JSON example for consistency